src/components/Education.tsx

```diff
@@ -188,7 +189,7 @@ const Education = () => {
     const handleUpdate = (id:any) => {
     
         axios
-            .post("http://localhost:8081/education/"+ id, {
+            .post(url + "/education/"+ id, {
```

Comment:
@ddinh please store url + "/education" to a variable and pass here to minimize code duplication